### PR TITLE
Kubernetes installation instructions rework

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: plugins/cilium-cni/cni-install.sh [hunk 1]
```diff
@@ -6,12 +6,17 @@ HOST_PREFIX=${HOST_PREFIX:-/host}
...
 
-CILIUM_CNI=${CILIUM_CNI:-${HOST_PREFIX}/opt/cni/bin/cilium-cni}
+CNI_DIR=${CNI_BINDIR:-${HOST_PREFIX}/opt/cni}
 CILIUM_CNI_CONF=${CILIUM_CNI_CONF:-${HOST_PREFIX}/etc/cni/net.d/${CNI_CONF_NAME}}
 
-# Install cilium-cni binary tohost
-echo "Installing $CILIUM_CNI ..."
-cp /opt/cni/bin/cilium-cni ${CILIUM_CNI}
+echo "Installing CNI base..."
+wget -nv https://storage.googleapis.com/kubernetes-release/network-plugins/cni-07a8a28637e97b22eb8dfe710eeae1344f69d16e.tar.gz
```
Comment (aanm) [inline plugins/cilium-cni/cni-install.sh]: The latest one is `network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz`

There's also a amd64 version `network-plugins/cni-amd64-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz`

File: examples/getting-started/Vagrantfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ Vagrant.require_version ">= 1.8.3"
 num_nodes = (ENV['NNODES'] || 1).to_i
 cilium_version = (ENV['CILIUM_VERSION'] || "v0.9.0-rc1")
 cilium_opts = (ENV['CILIUM_OPTS'] || "--labels id --kvstore consul --kvstore-opt consul.address 192.168.33.11:8500 -t vxlan")
```
Comment (aanm) [inline examples/getting-started/Vagrantfile]: `--labels id` should be removed?

**Review by aanm — COMMENTED** [review]
Until 1cc3bd59b6e192f3ec1049ed0b99330fd40d08b8 LGTM.

@tgraf I'm confused, if we filter all labels by default, what's the purpose of the `--labels` flag?

**Comment by tgraf** [comment]
> @tgraf I'm confused, if we filter all labels by default, what's the purpose of the --labels flag?

This was just a rebase on top of a pervious all-labels branch for testing. I will rebase this.

**Comment by tgraf** [comment]
I've singled out the CNI install changes. I will do a separate PR for them.

**Pull request merged**